ansible/roles/networking/tasks/main.yaml

```diff
@@ -36,6 +36,16 @@
       state: present
       reload: yes
 
+  - name: Flush Iptable rules
+    iptables:
+      table: "{{ item }}"
+      flush: true
+    with_items:
+      - nat
+      - raw
```

Comment:
From this convo https://mesosphere.slack.com/archives/C01MPKVBB5E/p1639503431000900?thread_ts=1639502917.498900&cid=C01MPKVBB5E we need to flush all tables, not just `nat` and `raw`.